lib/ansible/module_utils/vmware.py

```diff
@@ -1110,3 +1099,32 @@ def find_datastore_cluster_by_name(self, datastore_cluster_name):
             if dsc.name == datastore_cluster_name:
                 return dsc
         return None
+
+    @staticmethod
+    def wait_for_task(task, max_backoff=64, timeout=3600):
+        """
+        Function to wait for given task using exponential backoff algorithm
+        Args:
+            task: VMware task object
+            max_backoff: Maximum amount of sleep time in seconds
+            timeout: Timeout for the given task in seconds
+
+        Returns: TaskError for failure or True with result for successful task.
+        """
+        previous_failure = 0
```

Comment:
I'd call it a failure counter. I didn't get the meaning of this var while scanning for the first time.

Reply:
OK.